calabash-cucumber/lib/calabash-cucumber/store/preferences.rb

```diff
@@ -13,14 +14,80 @@ def initialize
         @path = File.join(dot_dir, "preferences", "preferences.json")
       end
 
+      # !@visibility private
+      def usage_tracking
+        preferences = read
+
+        unless valid_user_tracking_value?(preferences[:usage_tracking])
+          log_defaults_reset
+          preferences[:usage_tracking] = defaults[:usage_tracking]
+          write(preferences)
+        end
+
+        preferences[:usage_tracking]
+      end
+
+      # !@visibility private
+      def usage_tracking=(value)
+        if !valid_user_tracking_value?(value)
+          raise ArgumentError,
+            "Expected '#{value}' to be one of #{VALID_USAGE_TRACKING_VALUES.join(", ")}"
+        end
+
+        preferences = read
+        preferences[:usage_tracking] = value
+        write(preferences)
+      end
```

Comment:
Small steps.  I don't feel like I know enough about this interface to make a decision like this.  Once we have more than 2 keys in preferences, we should consider a generic solution.